app/export.go

```diff
@@ -62,6 +65,62 @@ var exportablePreferences = map[ComparablePreference]string{{
 }: "EmailInterval",
 }
 
+func (a *App) GetID() string {
+	return a.ID
+}
+func (a *App) SetID(id string) {
+	a.ID = id
+}
+
+func exportLogFile(attachment AttachmentImportData, exportFilename string) (*LogFormat, *model.AppError) {
+	createdAtTime, err := strconv.ParseInt(strconv.FormatInt(attachment.CreateAt, 10), 10, 64)
+	if err != nil {
+		return nil, model.NewAppError("parseTime", "Failed to parse createdAt time", nil, err.Error(), http.StatusInternalServerError)
+	}
+	updatedAtTime, err := strconv.ParseInt(strconv.FormatInt(attachment.UpdateAt, 10), 10, 64)
+	if err != nil {
+		return nil, model.NewAppError("parseTime", "Failed to parse updatedAt time", nil, err.Error(), http.StatusInternalServerError)
+	}
+	data := LogFormat{
+		Id:            attachment.Id,
+		CreatorId:     attachment.CreatorId,
+		PostId:        attachment.PostId,
+		ChannelId:     attachment.ChannelId,
+		CreateAt:      time.UnixMilli(createdAtTime),
+		UpdateAt:      time.UnixMilli(updatedAtTime),
```

Comment:
Why aren't you directly using `attachment.CreateAt` here?

Reply:
`attachment.CreateAt` is in integer form so i have to convert it into `time.Time`

Reply:
Done